src/main/java/world/bentobox/bentobox/managers/HooksManager.java

```diff
@@ -1,35 +1,50 @@
 package world.bentobox.bentobox.managers;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
 import org.eclipse.jdt.annotation.NonNull;
-
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Minor issue, but we use org.eclipse.jdt.annotation.NonNull and Nullable, not jet brains.

Reply:
It has been fixed, please review.